src/api_authentication_app_skeleton/src/actions/mixins/auth/action_helpers.cr

```diff
@@ -0,0 +1,30 @@
+require "jwt"
+
+module Auth::ActionHelpers
```

Comment:
This could probably be namespaced too since it is API specific. Otherwise it will clobber the browser-based auth helpers